src/style.css

```
@@ -0,0 +1,39 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  width: 96px;
+  justify-content: space-between;
+}
+
+.stars:not(:last-child) {
+  margin-bottom: 4px;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  background: url("images/star.svg") center no-repeat;
+}
+
+.stars--1 > .stars__star:nth-child(-1n + 1) {
+  background: url("images/star-active.svg") center no-repeat;
+}
+
+.stars--2 > .stars__star:nth-child(-1n + 2) {
+  background: url("images/star-active.svg") center no-repeat;
+}
+
+.stars--3 > .stars__star:nth-child(-1n + 3) {
+  background: url("images/star-active.svg") center no-repeat;
+}
+
+.stars--4 > .stars__star:nth-child(-1n + 4) {
+  background: url("images/star-active.svg") center no-repeat;
+}
+
+.stars--5 .stars__star:nth-child(-1n + 5) {
+  background: url("images/star-active.svg") center no-repeat;
+}
```

```suggestion
.stars--4 > .stars__star:nth-child(-n + 4),
.stars--5 .stars__star:nth-child(-n + 5) {
  background: url("images/star-active.svg");
}
```
ставити 1 перед n необовязково
запиши класи через кому
center i no-repeat краще прописати по дефолту ці стилі потірбні і активним і не активним зірочкам

---

Здравствуйте! По поводу по дефолту у меня выходит только так:
.stars__star,
.stars--1 > .stars__star:nth-child(-n + 1),
.stars--2 > .stars__star:nth-child(-n + 2),
.stars--3 > .stars__star:nth-child(-n + 3),
.stars--4 > .stars__star:nth-child(-n + 4),
.stars--5 > .stars__star:nth-child(-n + 5) {
  background-position: center;
  background-repeat: no-repeat;
}